README.md

```diff
@@ -0,0 +1,35 @@
+# ElasticSearch on Rancher
+
+Configuration for deploying the following services to [Rancher]:
+
+- `es-master`, ElasticSearch cluster, running on every host with a `escluster` tag defined.
+- Justwatch [Exporter], exposing Prometheus metrics on the cluster status.
+- es-client, Rancher cattle DNS service, which points the `es-client` domain name to `es-master`, for backwards compatibility with existing services.
+
+## Deployment
+
+Deployment is driven by Travis (Dev and PR review deployments) and TeamCity (staging and production deployments). We use the NHSuk [CI-Deployment] repo for deployment to the Rancher environment, this sets the following ENV variables.
+
+## Env vars
+
+| Variable | Description | Default | 
+| -------- | ----------- | ------- |
+| `ES_MIN_MASTER_NODES` | The minimum nodes required to run the host  | `2` |
```

Comment:
You mention de default to be 2, and lower down to be 3. Which one is it?

Reply:
@beseven That's correct.

See the documentation here: https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-node.html#split-brain

The aim is that there's 3 *eligible* master nodes, and this ENV variable is for the *minimum* number of nodes needed.